storage/aptosdb/src/aptossum/mod.rs

```diff
@@ -1,105 +0,0 @@
-// Copyright (c) Aptos
```

Comment:
weird... did you pull the latest?

Reply:
Yes, this is the latest. I think https://github.com/aptos-labs/aptos-core/pull/714 missed to remove this file.

Reply:
Shit, my bad~